.github/workflows/macos.yml

```diff
@@ -0,0 +1,72 @@
+name: "macOS builds"
+on: push
+jobs:
+  build_macos_clang:
+    name: "Clang (macOS-10.14)"
+    runs-on: macOS-10.14
+    steps:
+      - uses: actions/checkout@v1
+      - name: "Log environment"
+        run: ./scripts/log-env.sh
+      - name: Install dependencies
+        run: brew install $(./scripts/list-build-dependencies.sh --compiler clang)
+      - name: Build
+        run: ./scripts/build.sh --compiler clang --lto --fdo
+      - name: "Summarize warnings"
+        run: python3 ./scripts/count-warnings.py build.log
```

Comment:
Thanks; I caught the last one in macos.yml; pushing and testing now.

Reply:
/Users/runner/runners/2.159.2/work/_temp/07a15170-f7bc-4b43-8040-eab14682dd52.sh: ./scripts/count-warnings.py: /usr/bin/python3: bad interpreter: No such file or directory

Apple.. causing unnecessary head-aches :face_with_head_bandage: 
Will revert.

Reply:
Yeah, it's surprising… but that explains why I still see users on StackOverflow recommending hashbang `#!/usr/bin/env python3`. (Such hashbang caused issues to Linux distributions during python2 -> python3 transition and I personally saw it being a cause of user-bugs that couldn't be detected during testing by software developers - but it might have valid usecases in some contexts).